src/Model/Command/ItemPropertySetup.php

```diff
@@ -60,6 +60,11 @@ public static function set(string $propertyName): self
     protected function setPropertyName(string $propertyName): void
     {
         Assertion::typeIdentifier($propertyName);
+        Assertion::notEq(
+            $propertyName,
+            'item_id',
+            'Cannot redefine property item_id - it is used as primary key to identify items. Please use different name.'
```

Comment:
`Cannot manipulate with property "item_id" - it is used by Matej to identify items.` or something?